rust/src/controllers/telemetry.rs

```diff
@@ -419,21 +419,6 @@ pub fn username_has_follower(
     Ok(!rows.is_empty())
 }
 
-pub fn sync_users_location(
```

Comment:
Why did you remove this function instead of refactoring it? @darioalessandro

Reply:
Hi @JasterV:

so it turns out that all we needed to do is call force refresh for the invitation creator